how-it-works.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
This is another problem, try scrolling sideways, maybe reduce the size of the images
![image](https://user-images.githubusercontent.com/60205816/182843811-1cac80e7-2698-49de-8599-4a82acb3c63a.png)